script/rubocop-autocorrect.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+#
+# Fixes safe cops automatically and creates a commit for each.
+#
+
+if git add --dry-run --all | grep --quiet .; then
+  echo "Dirty working tree. Please start on a fresh branch."
+  exit 1
+fi
+
+# Refresh todo file so that following commits include only related changes:
+bundle exec rubocop --regenerate-todo
+git commit --all --message "Regenerate Rubocop's TODO file"
+
+# Iterate over all safe cops:
+grep "This cop supports safe autocorrection" -A 5 .rubocop_todo.yml\
+  | grep '^[A-Z]'\
+  | tr -d :\
+  | while read cop; do
```

Comment:
I was experimenting with adding `head -n 3` or `grep "^Style"` as additional filters. We may want those as command line options but I kept it simple for now.